mmdet/core/data_structures/general_data.py

```diff
@@ -273,6 +273,16 @@ def cpu(self):
             new_data[k] = v
         return new_data
 
+    # Tensor-like methods
+    def mlu(self):
```

Comment:
can we also allow the usage like tensor.to('mlu')?

Reply:
I don't quite understand what you mean. Can you explain it in detail.

Reply:
> can we also allow the usage like tensor.to('mlu')?

yes we support the usage tensor.to('mlu'), and  the Tensor-like method "mlu" here refer to the Tensor-like method "cuda" below .